Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -1195,9 +1195,11 @@
     <Word from="SoBl3lNG" to="SOBBING" />
     <Word from="Sojust" to="So just" />
     <Word from="soldierl" to="soldier!" />
-    <Word from="somethlng" to="somethlng" />
+    <Word from="somethlng" to="something" />
     <Word from="somethlng's" to="something's" />
     <Word from="somez'/7/ng" to="something" />
+    <Word from="somthing" to="something" />
++   <Word from="sumthing" to="something" />

```

Comment:
I'm afraid, XmlDocument.Load() (see [Logic\Dictionaries\OcrFixReplaceList.cs](https://github.com/SubtitleEdit/subtitleedit/blob/master/src/Logic/Dictionaries/OcrFixReplaceList.cs#L786)) doesn't like the '+' at the start of line 1202.